ols/app/endpoints/feedback.py

```diff
@@ -136,7 +140,8 @@ def store_user_feedback(
             detail={
                 "response": "Error storing user feedback",
                 "cause": str(e),
+                "timestamp": datetime.utcnow().strftime('%Y-%m-%d %H:%M:%S'),
```

Comment:
not strictly needed - we need to store timestamp into JSON file, not to provide it in response (some timestamp is there already in HTTP header). But I guess it does not do any harm.